cognite/powerops/resync/config/_main.py

```diff
@@ -197,9 +201,10 @@ def validate_shop_related_files(cls, value):
 
     @classmethod
     def instantiate_from_dict(cls, config: dict) -> CogShopConfig:
-        time_series_mappings_v2 = []
-        for time_series_mapping in config["time_series_mappings_v2"]:
-            time_series_mappings_v2.append(TimeSeriesMappingV2.load_from_dict(time_series_mapping))
```

Comment:
mapping removed ?

Reply:
just rewrote the for loop into a list comprehension